### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by hritik5102**
@emilsoman please check the updated documentation ( issue #2 )

**Content** 

- [x] **_Necessary changes in readme.md_**
- [x] **_CODE_OF_CONDUCT.md_**
- [x] **_.gitignore_**
- [x] **_Contributing.md_**
- [x] **_Contributors.md_**

If any changes required please let me know :)
Thank you

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+<img align="right" width="300" src="assets/images/fork.png" alt="fork this repository" />
+
+## Fork this repository
+
+Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+
+## Clone the repository
+
+<img align="right" width="300" src="assets/images/clone.png" alt="clone this repository" />
```
Comment (polvalente) [inline CONTRIBUTING.md:21]: I guess the PNG images make this Markdown less mobile friendly, specially since they have fixed width. Could we remove them?

Comment (polvalente) [reply CONTRIBUTING.md:21]: Maybe add hyperlinks to the respective GitHub documentation page

File: README.md [hunk 2]
```diff
@@ -1,17 +1,32 @@
...
 Fika is designed for building and maintaining scalable web apps without
 compromising on programmer ergonomics.
 
-Project status: Actively developed. Not ready for production use or HackerNews.
+**Project status**: Actively developed. Not ready for production use or HackerNews.
+
+### Clone git repository
+
+```sh
+    $ git clone "https://github.com/fika-lang/fika"
```
Comment (polvalente) [inline README.md:24]: Is there a reason for the indentation?

**Comment by emilsoman** [comment]
Sorry @hritik5102 , unfortunately I'll have to turn down this PR again because its scope is too big yet again. I hope you got a chance to read the issue and the comments I left before re-working. Thank you for your effort though!

**Pull request closed without merging**